cloud-storage-uploader/tests/upload-helper.int.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import 'mocha';
+import { UploadHelper } from '../src/upload-helper';
+import { Storage } from '@google-cloud/storage';
+
+const testBucket = process.env.TEST_BUCKET;
```

Comment:
Make the test variables scoped to the action name, so, e.g. `UPLOAD_CLOUD_STORAGE_TEST_BUCKET`

Reply:
Done